Sources/Tropos/Views/DailyForecastView.swift

```diff
@@ -0,0 +1,27 @@
+import UIKit
+import TroposCore
+
+@objc(TRDailyForecastView) public class DailyForecastView: UIView {
+    @IBOutlet weak var dayOfWeekLabel: UILabel!
+    @IBOutlet weak var highTemperatureLabel: UILabel!
+    @IBOutlet weak var lowTemperatureLabel: UILabel!
+    @IBOutlet weak var conditionsImageView: UIImageView!
+    @IBOutlet var contentView: DailyForecastView!
+    @objc public var viewModel: DailyForecastViewModel!
+
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+    }
+
+    public override func awakeFromNib() {
+        super.awakeFromNib()
+        let mainBundle = Bundle.main
+        mainBundle.loadNibNamed(String(describing: "TRDailyForecastView"), owner: self, options: nil)
```

Comment:
```suggestion
        mainBundle.loadNibNamed(String(describing: "TRDailyForecastView"), owner: self)
```

Does this compile if you leave off the `options` parameter? I _think_ it might have a default value of `nil`.

Reply:
Leaving out options worked.